apps/fluent-tester/src/TestComponents/Overflow/OverflowTest.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { ButtonV1 as Button } from '@fluentui-react-native/button';
+import { Divider } from '@fluentui-react-native/divider';
+import { Menu, MenuTrigger, MenuPopover, MenuItem } from '@fluentui-react-native/menu';
+import { Overflow, OverflowItem, useOverflowMenu } from '@fluentui-react-native/overflow';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+
+import MoreHorizontalIcon from './MoreHorizontalFilled.svg';
+import { Test } from '../Test';
+
+const items = ['a', 'b', 'c', 'd', 'e'];
+const itemLabels = {
+  a: 'Item A',
+  b: 'Item B Item B Item B',
+  c: 'Item C Item C',
+  d: 'Item D',
+  e: 'Item E Item E Item E Item E',
+};
+
+const overflowTestPageStyles = StyleSheet.create({
+  containerStyle: {
+    paddingVertical: 4,
+  },
+  width100: {
+    width: 100,
+  },
+  width250: {
+    width: 250,
+  },
+  width400: {
+    width: 400,
+  },
+});
+
+interface OverflowMenuProps {
+  onItemPress: (id: string) => void;
+}
+
+function OverflowMenu(props: OverflowMenuProps) {
+  const { showMenu, menuItemVisibilities, menuRef, onLayout } = useOverflowMenu();
+  if (showMenu) {
+    return (
+      <Menu>
+        <MenuTrigger>
+          <Button
+            accessibilityLabel="More options"
+            onLayout={onLayout}
+            style={{ alignSelf: 'center' }}
+            appearance="subtle"
+            iconOnly
+            icon={{ svgSource: { src: MoreHorizontalIcon, viewBox: '0 0 20 20' } }}
+            componentRef={menuRef}
+          />
+        </MenuTrigger>
+        <MenuPopover>
+          {items
+            .filter((id) => menuItemVisibilities[id])
+            .map((id) => (
+              <MenuItem onClick={() => props.onItemPress(id)} key={id}>
+                {itemLabels[id]}
+              </MenuItem>
+            ))}
+        </MenuPopover>
+      </Menu>
+    );
+  } else {
+    return null;
+  }
+}
+
+export function OverflowMainTest() {
+  return (
+    <View style={overflowTestPageStyles.containerStyle}>
+      <Overflow itemIDs={items}>
+        {items.map((item) => (
+          <OverflowItem key={item} overflowID={item}>
+            <Button>{itemLabels[item]}</Button>
+          </OverflowItem>
+        ))}
+        <OverflowMenu onItemPress={console.log} />
+      </Overflow>
+    </View>
+  );
+}
+
+export function OverflowDifferentWidthTest() {
+  return (
+    <View>
+      <Text variant="heroStandard">Hidden Before Layout</Text>
+      <View style={overflowTestPageStyles.containerStyle}>
+        <Text variant="body1Strong">Width: 400</Text>
+        <Overflow style={overflowTestPageStyles.width400} itemIDs={items}>
+          {items.map((item) => (
+            <OverflowItem overflowID={item} key={item}>
+              <Button>Item {' ' + item}</Button>
+            </OverflowItem>
+          ))}
+          <OverflowMenu onItemPress={console.log} />
+        </Overflow>
```

Comment:
Seems like this could be its own component to reduce code copying.

Reply:
The web equivalent Overflow doesn't export an OverflowMenu, relying on users to implement their own dropdown menu using `useOverflowMenu`. Do you want to diverge on that?

Reply:
No, I meant that you could refactor out a component for the Test file. Lines 94-101 are copied below with width being the only difference, that could be made into a component with items and width as props.